<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MshQualitySelection</class>
 <widget class="QDialog" name="MshQualitySelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesh Quality Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select Quality Measure</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="choiceEdges">
        <property name="text">
         <string>Aspect Ratio of Edge Length</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="choiceArea">
        <property name="text">
         <string>Area of 2D Elements</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="choiceVolume">
        <property name="text">
         <string>Volume of 3D Elements</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="choiceAngles">
        <property name="text">
         <string>Angles between Adjacent Edges</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MshQualitySelection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MshQualitySelection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
